notary-commons/src/main/kotlin/util/FileUtil.kt

```diff
@@ -0,0 +1,16 @@
+package util
+
+import java.io.File
+import java.io.IOException
+
+/**
+ * Creates folder with all parent sub-folders.
+ * @param folderPath - path of folder to create
+ * @throws IOException, if it's impossible to create folder
+ */
+fun createFolderIfDoesntExist(folderPath: String) {
+    val folder = File(folderPath)
+    if (!folder.exists() && !File(folderPath).mkdirs()) {
```

Comment:
`mkdirs()` `Creates the directory named by this abstract pathname, including any necessary but nonexistent parent directories. `
Can't we just have `if(!folder.mkdirs())` ?

Reply:
No, because this function returs false, if folder exists.